tests/v2/integration/cli/test_cli.py

```diff
@@ -29,6 +29,7 @@ def test_otx_cli(task: str, tmp_dir_path: Path) -> None:
     """
     # 1. Training
     tmp_dir_path = tmp_dir_path / f"{task}_e2e_test"
+    deterministic = "False" if task == "segmentation" else "True"
```

Comment:
Is there any special reason to set False only for segmentation?

Reply:
upsample_bilinear2d_backward_out_cuda does not support deterministic
`RuntimeError: upsample_bilinear2d_backward_out_cuda does not have a deterministic implementation, but you set \'torch.use_deterministic_algorithms(True)\'. You can turn off determinism just for this operation, or you can use the \'warn_only=True\' option, if \nthat\'s acceptable for your application. You can also file an issue at https://github.com/pytorch/pytorch/issues to help us prioritize adding deterministic support for this operation.\n'))`

Reply:
thank you. let's keep thread in Jaeguk's comment.